builtin/logical/transit/path_encrypt.go

```diff
@@ -159,7 +159,7 @@ func decodeBatchRequestItems(src interface{}, dst *[]BatchRequestItem) error {
 			} else if casted, ok := v.(string); ok {
 				(*dst)[i].Context = casted
 			} else {
-				errs.Errors = append(errs.Errors, fmt.Sprintf("'[%d].context' expected type 'string', got unconvertible type '%T'", i, item["context"]))
+				errs.Errors = append(errs.Errors, fmt.Sprintf("'[%d].context' expected type 'string', got unconvertible type '%T', value: '%v'", i, item["context"], item["context"]))
```

Comment:
How was this necessary for bumping dependencies?

Reply:
TestTransit_decodeBatchRequestItems has the following check: 

`			expectedErr := mapstructure.Decode(tt.src, &expectedDest)

			gotErr := decodeBatchRequestItems(tt.src, &tt.dest)
			gotDest := tt.dest

			if !reflect.DeepEqual(expectedErr, gotErr) {
				t.Errorf("decodeBatchRequestItems unexpected error value, want: '%v', got: '%v'", expectedErr, gotErr)
			}`

which fails without these changes like so: https://app.circleci.com/pipelines/github/hashicorp/vault/15230/workflows/0b4e1411-9665-4b36-9991-e6cac823a4ac/jobs/149962

Since the comment above decodeBatchRequestItems says: 

`
// It aims to behave as closely possible to the original mapstructure.Decode 
`

I figured I should make the two error messages match instead of changing the test.

Reply:
Fair enough.